metaflow/plugins/aws/step_functions/step_functions_client.py

```diff
@@ -82,8 +82,18 @@ def list_executions(self, state_machine_arn, states):
         )
 
     def terminate_execution(self, state_machine_arn, execution_arn):
-        # TODO
-        pass
+        try:
```

Comment:
this still needs a counterpart in the `step_functions_cli.py` so the function can be called through the CLI. You can see how the command has been implemented in `argo_workflows_cli.py` for some example on the plumbing.

Another thing that needs to be implemented for this feature is a `validate_run_id` (again, similar to argo CLI) which checks that the run_id the user has provided belongs to the correct flow/project/user and has a matching token.
This is to guard against being able to issue arbitrary commands such as
```sh
python anyflow.py step-functions terminate not-my-run-id
```